packages/ng/comment/comment-chat/comment-chat.component.html

```diff
@@ -0,0 +1,3 @@
+<ol class="commentWrapperChat">
```

Comment:
I wonder if this shouldn't be considered as a new component as I heard Clément speaking about "Chatbox". I'll investigate.